scripts/system/dialog.js

```diff
@@ -315,9 +323,7 @@ export class CombatDialog extends RollDialog {
             defence : 3,
             armour : 0
         }
-        data.secondaryTarget = duplicate(data.primaryTarget)
```

Comment:
I thought this was needed, why was it removed?

Reply:
May have been some over zealous refactoring it was indeed erroneously removed